doc/api/esm.md

```diff
@@ -391,6 +391,33 @@ import { readFileSync } from 'node:fs';
 const buffer = readFileSync(new URL('./data.proto', import.meta.url));
 ```
 
+### `import.meta.main`
+
+<!-- YAML
+added:
+  - REPLACEME
```

Comment:
```suggestion
added: REPLACEME
```

Reply:
I’d keep it as is, we probably want to backport it